discord/webhook.py

```diff
@@ -1,28 +1,32 @@
 from .embeds import Embed
+from .mixins import Hashable
 from .errors import HTTPException
 from .utils import _bytes_to_base64_data
 import aiohttp
 import io
 import json
 import asyncio
+
+# Webhook code made by Cerulean#7014 
+
 WEBHOOK_BASE = 'https://discordapp.com/api/webhooks/'
-class Webhook():
-    def __init__(self,**kwargs):
-        self.avatar = kwargs.get('avatar','')
-        self.name = kwargs.get('name','')
+AVATAR_BASE = 'https://cdn.discordapp.com/avatars/{id}/{avatar}.png'
+class Webhook(Hashable):
+    __slots__ = ( 'name', 'id', 'avatar', 'url', '_state',
+                 'webhook', 'token')
+    def __init__(self,state, **kwargs):
+        self._state = state
+        
+        self.name = kwargs.get('name',None)

```

Comment:
`None` is unnecessary, as it will be automatically returned if the key is not found.